Projects/App/Sources/Driver/Service/GifticonAPI.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  GifticonAPI.swift
+//  GGiriGGiri
+//
+//  Created by AhnSangHoon on 2022/07/04.
+//  Copyright © 2022 dvHuni. All rights reserved.
+//
+
+import Foundation
+
+enum GifticonAPI {
+    case categoryList(GifticonListRquestModel)
```

Comment:
```suggestion
    case categoryList(GifticonListRequestModel)
```
요거 스펠링 수정이 필요할거같습니다!